classes/patterns/class-deepdivetopic.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Deep Dive Topic pattern class.
+ *
+ * @package P4GBKS
+ */
+
+namespace P4GBKS\Patterns;
+
+use P4GBKS\Patterns\Templates\Covers;
+
+/**
+ * Class Deep Dive Topic.
+ *
+ * @package P4GBKS\Patterns
+ */
+class DeepDiveTopic extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/deep-dive-topic-pattern-layout';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 *
+	 * @param array $params Optional array of parameters for the config.
+	 */
+	public static function get_config( $params = [] ): array {
+		$classname = self::get_classname();
+
+		return [
+			'title'      => __( 'Deep Dive Topic', 'planet4-blocks-backend' ),
+			'categories' => [ 'layouts' ],
+			'blockTypes' => [ 'core/post-content' ],
+			'content'    => '
+				<!-- wp:group {"className":"block ' . $classname . '"} -->
+					<div class="wp-block-group ' . $classname . '">
+					' . PageHeader::get_config(
```

Comment:
On the mockups I see that the PageHeader has a grey background, you'll need [this PR](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/926) to be merged in order to add it 🙂